_docs/credentials.md

```diff
@@ -123,6 +125,20 @@ or
 
 	$ gradle showCredentials -Plocation=home
 
+#### ***Status***
+
+It is also possible to display the status of all credentials(*valid/invalid*) by the parameter ***status***
+
+	$ gradle showCredentials -Pstatus
+
+This parameter can filter credentials with two values: 'isValid' and 'isInvalid'.
```

Comment:
use simple words..

This parameter can filter credentials with two values: 'isValid' and 'isInvalid'.

This parameter can filter credentials with two values: 'valid' and 'invalid'.